### PR TITLE
Use random iv

### PR DESCRIPTION
**Pull request by mehrdada**

### CODE REVIEW

File: AuthenticationExample.Web/AuthenticationExample.Web.csproj [hunk 1]
```diff
@@ -79,7 +79,9 @@
     <Compile Include="ViewModels\UserInputModel.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="packages.config" />
```
Comment (friism) [inline AuthenticationExample.Web/AuthenticationExample.Web.csproj:82]: this change seems unrelated

File: AppHarbor.Web.Security/SymmetricEncryption.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System.IO;
 using System.Security.Cryptography;
+using System;
```
Comment (friism) [inline AppHarbor.Web.Security/SymmetricEncryption.cs]: order usings

File: AuthenticationExample.Web/AuthenticationExample.Web.csproj (hunk 1 above)
Comment (mehrdada) [inline AuthenticationExample.Web/AuthenticationExample.Web.csproj:82]: I don't know where this came from. Will remove
